lighthouse-core/report/v2/report-template.html

```diff
@@ -32,7 +32,8 @@
   <script>
     const dom = new DOM(document);
     const detailsRenderer = new DetailsRenderer(dom);
-    const renderer = new ReportRenderer(dom, detailsRenderer);
+    const categoryRenderer = new CategoryRenderer(dom, detailsRenderer);
```

Comment:
ha, ya know I hear there's some newfangled way these days to state that a file `require`'s another...I wonder if that's something we could look into?